src/Corax/IndexWriter.cs

```diff
@@ -1071,8 +1071,7 @@ private void RemoveSingleTerm(IndexedField indexedField, IndexEntryReader.FieldR
 
             void RecordTupleToDelete(IndexedField indexedField, ReadOnlySpan<byte> termValue, double termDouble, long termLong)
             {
-                // Is there any reason to analyze string of number?
-                RecordExactTermToDelete(termValue, indexedField);
+                RecordTermToDelete(termValue, indexedField);
```

Comment:
But dates are _not_ being analyzed, at least they are not supposed to.
They are indexed exactly.